### PR TITLE
docs: enable policy enable/disable help text.

### PR DESCRIPTION
**Pull request by dmurray-lacework**
Signed-off-by: Darren Murray <darren.murray@lacework.net>

## Summary

Enable help text for: 
lacework policy enable [policy-id] --tag [tag]
lacework policy disable [policy-id] --tag [tag]

## How did you test this change?

Run help text tests

### CODE REVIEW

File: integration/test_resources/help/policy_disable
Comment (afiune) [inline integration/test_resources/help/policy_disable:10]: All of this text needs to be updated to render Markdown correctly, you have to add an empty line in between regular text and "code text".

These instructions will look like this in Markdown:

Screen Shot 2022-04-13 at 3 00 13 PM

But if you add a carrier return in between, it will show it as code:

Screen Shot 2022-04-13 at 3 00 32 PM

Please update all texts similar to this one.

Comment (afiune) [reply integration/test_resources/help/policy_disable:10]: So, this should be looking like this:
```suggestion
To disable a single policy by it's ID:

	lacework policy disable lacework-policy-id

To disable all policies for Aws CIS 1.4.0:

	lacework policy disable --tag framework:cis-aws-1-4-0

To disable all policies for Gcp CIS 1.3.0:

	lacework policy disable --tag framework:cis-gcp-1-3-0
```

**Review by afiune — CHANGES_REQUESTED** [review]

File: integration/test_resources/help/policy_disable
Comment (dmurray-lacework) [reply integration/test_resources/help/policy_disable]: fixed

File: cli/cmd/compliance_gcp.go
Comment (afiune) [inline cli/cmd/compliance_gcp.go:374]: are you planning to update all other compliance commands? or wait until we "again" review and unhide them?

I would appreciate if we update all of them but it is your call, as long as you remember to do so.

**Review by afiune — APPROVED** [review]

File: cli/cmd/compliance_gcp.go
Comment (dmurray-lacework) [reply cli/cmd/compliance_gcp.go:374]: I had not planned on changes to other compliance commands in this PR. It would make sense to allow the CIS version to be an accepted value.

Comment (dmurray-lacework) [reply cli/cmd/compliance_gcp.go:374]: Created a ticket to tackle this at a later date -> https://lacework.atlassian.net/browse/ALLY-982

**Pull request merged**
